### PR TITLE
Viridian -> Cloud

### PR DESCRIPTION
**Pull request by rebekah-lawrence**
# Description of change

Changes to top-level playbooks, addition of playbooks for 5.4, and other changes on this repo to README and search-config to support the change of name from Viridian to Cloud

## Type of change

Select the type of change that you're making:

- [ ] Bug fix (Addresses an issue in existing content such as a typo)
- [ ] Enhancement (Adds new content)
- [x] rename of Viridian to cloud

## Open Questions and Pre-Merge TODOs
- [ ] Use github checklists to create a list. When an item is solved, check the box and explain the answer.

### CODE REVIEW

**Review by oliverhowell — APPROVED** [review]
LGTM (conflict resolution aside)

File: antora-playbook-local.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ content:
   - url: https://github.com/hazelcast-guides/airline-connections
     branches: main
     start_path: docs
-  - url: https://github.com/hazelcast-guides/sql_basics_on_viridian
+  - url: https://github.com/hazelcast-guides/sql_basics_on_cloud
```
Comment (Serdaro) [inline antora-playbook-local.yml:38]:
```suggestion
  - url: https://github.com/hazelcast-guides/sql_stock_ticker_cloud
```

File: antora-playbook.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ content:
   - url: https://github.com/hazelcast-guides/Generating-Data-using-SQL
     branches: master
     start_path: docs
-  - url: https://github.com/hazelcast-guides/sql_basics_on_viridian
+  - url: https://github.com/hazelcast-guides/sql_basics_on_cloud
```
Comment (Serdaro) [inline antora-playbook.yml:44]:
```suggestion
  - url: https://github.com/hazelcast-guides/sql_stock_ticker_cloud
```

File: assembler-playbook-platform-5-4.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+site:
+  title: Documentation
+  url: https:/docs.hazelcast.com
+content:
+  sources: 
+  - url: https://github.com/serdaro/hz-docs
+    branches: [54-pdf]
```
Comment (Serdaro) [inline assembler-playbook-platform-5-4.yml:7]:
```suggestion
    branches: [53-pdf]
```

**Review by Serdaro — APPROVED** [review]
Approved with minor fixes.
(let the 53-pdf for now stay as it is, as there is no 54-pdf branch on my profile).

**Pull request merged**